packages/editor/src/components/prefabs/PrefabEditors.tsx

```diff
@@ -48,7 +50,9 @@ export const PrefabIcons: Record<string, ReactNode> = {
   Video: <img src={VideoIcon} />,
   Lookdev: <img src={LookDevIcon} />,
   Camera: <img src={CameraIcon} />,
-  default: <img src={AddIcon} />
+  Empty: <img src={AddIcon} />,
+  Ecommerce: <img src={EcommIcon} />,
```

Comment:
ecommerce can't exist in the base engine. please move this to the appropriate project

Reply:
This was already included in the engine fyi. This is a bug fix. Working on a way to make projects expose prefab icons rn.